example_specs/models/unet2d_nuclei_broad/rdf_v0_1_0.yaml

```diff
@@ -0,0 +1,99 @@
+format_version: 0.1.0
+
+name: UNet 2D Nuclei Broad
+description: A 2d U-Net trained on the nuclei broad dataset.
+authors:
+  - "Constantin Pape;@bioimage-io"
+  - "Fynn Beuttenmueller"
+
+# we allow for multiple citations. Each citation contains TEXT, DOI and URL. One of DOI or URL needs to be given.
+cite:
+  - text: "Ronneberger, Olaf et al. U-net: Convolutional networks for biomedical image segmentation. MICCAI 2015."
+    doi: https://doi.org/10.1007/978-3-319-24574-4_28
+  - text: "2018 Data Science Bowl"
+    url: https://www.kaggle.com/c/data-science-bowl-2018
+
+tags: [unet2d, pytorch, nucleus, segmentation, dsb2018]
+license: MIT
+
+documentation: README.md
+covers: [cover0.png]
+
+inputs:
+  - name: raw
+    axes: bcyx  # letters of axes in btczyx
+    data_type: float32
+    data_range: [-.inf, .inf]
+    shape: [1, 1, 512, 512]
+
+outputs:
+  - name: probability
+    axes: bcyx
+    data_type: float32
+    data_range: [-.inf, .inf]
+    halo: [0, 0, 32, 32]
+    shape:
+      reference_input: raw
+      scale: [1.0, 1.0, 1.0, 1.0]
+      offset: [0, 0, 0, 0]
+
+language: python
+framework: pytorch
+source: unet2d.py::UNet2d
+optional_kwargs: {input_channels: 1, output_channels: 1}
+
+test_input: test_input.npy
+test_output: test_output.npy
+
+prediction:
+  dependencies: conda:./environment.yaml
+  weights:
+     source: https://zenodo.org/record/3446812/files/unet2d_weights.torch
+     hash: {sha256: e4d3885bccbe41cbf6c1d825f3cd2b707c7021ead5593156007e407a16b27cf2}
+
+config:
```

Comment:
I think it would be worth leaving a comment here, what this future stuff is about. `convert_model_from_v0_1` has no docstring, I guess these values are intended to be used there?

Reply:
yes, I will add documentation for this